com.hazelcast.cp.internal.CPMemberAddRemoveTest.when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot [HZ-1837]

_master_ (commit 285517dcd7b6c95a2d751f97140a1f046230adca)
Failed on oracle-11 (sonar build): https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-sonar/1372/testReport/junit/com.hazelcast.cp.internal/CPMemberAddRemoveTest/when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot/
<details><summary>Stacktrace:</summary>

```
java.util.concurrent.ExecutionException: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:121)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.resolveAndThrowIfException(InvocationFuture.java:100)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.get(AbstractInvocationFuture.java:617)
	at com.hazelcast.cp.internal.CPMemberAddRemoveTest.when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot(CPMemberAddRemoveTest.java:950)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
	at com.hazelcast.cp.internal.MetadataRaftGroupManager.checkMetadataGroupInitSuccessful(MetadataRaftGroupManager.java:1108)
	at com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp.run(GetActiveCPMembersOp.java:43)
	at com.hazelcast.cp.internal.raftop.metadata.MetadataRaftGroupOp.run(MetadataRaftGroupOp.java:38)
	at com.hazelcast.cp.internal.NodeEngineRaftIntegration.runOperation(NodeEngineRaftIntegration.java:219)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.applyLogEntry(RaftNodeImpl.java:849)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.applyLogEntries(RaftNodeImpl.java:792)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.commitEntries(RaftNodeImpl.java:1299)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.tryAdvanceCommitIndex(RaftNodeImpl.java:1283)
	at com.hazelcast.cp.internal.raft.impl.handler.AppendSuccessResponseHandlerTask.handleResponse(AppendSuccessResponseHandlerTask.java:71)
	at com.hazelcast.cp.internal.raft.impl.handler.AbstractResponseHandlerTask.innerRun(AbstractResponseHandlerTask.java:43)
	at com.hazelcast.cp.internal.raft.impl.task.RaftNodeStatusAwareTask.run(RaftNodeStatusAwareTask.java:52)
	at com.hazelcast.cp.internal.NodeEngineRaftIntegration.execute(NodeEngineRaftIntegration.java:114)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.execute(RaftNodeImpl.java:883)
	at com.hazelcast.cp.internal.raft.impl.RaftNodeImpl.handleAppendResponse(RaftNodeImpl.java:403)
	at com.hazelcast.cp.internal.RaftService.handleAppendResponse(RaftService.java:718)
	at com.hazelcast.cp.internal.operation.integration.AppendSuccessResponseOp.run(AppendSuccessResponseOp.java:48)
	at com.hazelcast.spi.impl.operationservice.Operation.call(Operation.java:190)
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:283)
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:258)
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:486)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:197)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:137)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.executeRun(OperationThread.java:123)
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:348)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:112)
	... 15 more

```
</details>
<details><summary>Standard output:</summary>

```
Finished Running Test: testCPMemberIdentityChanges_whenLocalMemberIsRecovered_duringRestart in 35.667 seconds.
03:04:23,228 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 100
03:04:23,229 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 100
03:04:23,229 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 100
03:04:25,737  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2512, invocationTime=1665975865736 (2022-10-17 03:04:25.736), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=120, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:25,974  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=232, invocationTime=1665975865972 (2022-10-17 03:04:25.972), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=120, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@68ce14c6}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:30,750  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2524, invocationTime=1665975870749 (2022-10-17 03:04:30.749), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=130, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:31,009  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=242, invocationTime=1665975871007 (2022-10-17 03:04:31.007), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=130, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@12006f98}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:35,809  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2534, invocationTime=1665975875792 (2022-10-17 03:04:35.792), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=140, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:36,040  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=252, invocationTime=1665975876029 (2022-10-17 03:04:36.029), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=140, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@63a379d0}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:40,820  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2544, invocationTime=1665975880817 (2022-10-17 03:04:40.817), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=150, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:41,068  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=262, invocationTime=1665975881066 (2022-10-17 03:04:41.066), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=150, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@33fc3b0d}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:45,840  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2556, invocationTime=1665975885839 (2022-10-17 03:04:45.839), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=160, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:46,089  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=272, invocationTime=1665975886087 (2022-10-17 03:04:46.087), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=160, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@2b0c6190}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:48,348 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 150
03:04:48,349 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 150
03:04:48,349 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 150
03:04:50,864  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2566, invocationTime=1665975890858 (2022-10-17 03:04:50.858), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=170, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:51,117  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=282, invocationTime=1665975891115 (2022-10-17 03:04:51.115), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=170, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@3a61b7c6}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:55,885  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2576, invocationTime=1665975895885 (2022-10-17 03:04:55.885), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=180, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:04:56,145  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=292, invocationTime=1665975896143 (2022-10-17 03:04:56.143), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=180, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@427424a3}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:00,914  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2588, invocationTime=1665975900910 (2022-10-17 03:05:00.910), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=190, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:01,169  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=302, invocationTime=1665975901168 (2022-10-17 03:05:01.168), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=190, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@10025bb3}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:05,945  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2598, invocationTime=1665975905944 (2022-10-17 03:05:05.944), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=200, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:06,199  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=312, invocationTime=1665975906197 (2022-10-17 03:05:06.197), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=200, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@584ab8c2}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:10,958  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2608, invocationTime=1665975910957 (2022-10-17 03:05:10.957), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=210, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:11,219  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=322, invocationTime=1665975911217 (2022-10-17 03:05:11.217), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=210, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@6fa2eb8a}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:13,478 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 200
03:05:13,478 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 200
03:05:13,479 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Taking snapshot for commit-index: 200
03:05:15,987  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2620, invocationTime=1665975915985 (2022-10-17 03:05:15.985), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=220, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:16,242  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=332, invocationTime=1665975916240 (2022-10-17 03:05:16.240), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=220, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@c108be1}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:21,015  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2630, invocationTime=1665975921014 (2022-10-17 03:05:21.014), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=230, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:21,273  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=342, invocationTime=1665975921271 (2022-10-17 03:05:21.271), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=230, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@ecd8613}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:26,033  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.DefaultRaftReplicateOp{serviceName='hz:core:raft', identityHash=1720844499, partitionId=0, replicaIndex=0, callId=2640, invocationTime=1665975926032 (2022-10-17 03:05:26.032), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, op=com.hazelcast.cp.internal.raftop.metadata.GetActiveCPMembersOp{serviceName='hz:core:raft'}}, tryCount=250, tryPauseMillis=500, invokeCount=240, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811913, firstInvocationTime='2022-10-17 03:03:31.913', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@381f2056}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:26,295  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Invocation] hz.suspicious_ardinghelli.response-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Retrying invocation: Invocation{op=com.hazelcast.cp.internal.operation.RaftQueryOp{serviceName='hz:core:raft', identityHash=1401333607, partitionId=0, replicaIndex=0, callId=352, invocationTime=1665975926294 (2022-10-17 03:05:26.294), waitTimeout=-1, callTimeout=120000, tenantControl=com.hazelcast.spi.impl.tenantcontrol.NoopTenantControl@0, op=com.hazelcast.cp.internal.raftop.metadata.GetRaftGroupOp{serviceName='hz:core:raft', targetGroupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}}, groupId=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, policy=LINEARIZABLE}, tryCount=250, tryPauseMillis=500, invokeCount=240, callTimeoutMillis=120000, firstInvocationTimeMs=1665975811912, firstInvocationTime='2022-10-17 03:03:31.912', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5701, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=com.hazelcast.internal.server.FirewallingServer$FirewallingConnection@34cb3a0}, Reason: com.hazelcast.cp.internal.exception.MetadataRaftGroupInitInProgressException
03:05:32,035 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is committed for CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701} with seed: 1665975810904 and discovered CP members: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,035 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Persisting active CP members [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}] with commitIndex 236
03:05:32,035 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.partition-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is completed with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}]
03:05:32,035  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.vibrant_engelbart.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] CP Subsystem is initialized with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,036  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.cached.thread-2 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] CP Subsystem is initialized with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,036  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.cached.thread-1 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] CP Subsystem is initialized with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is committed for CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701} with seed: 1665975810904 and discovered CP members: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is committed for CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701} with seed: 1665975810904 and discovered CP members: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}]
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Persisting active CP members [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}] with commitIndex 236
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Persisting active CP members [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}] with commitIndex 236
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.flamboyant_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is completed with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}]
03:05:32,036 DEBUG |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MetadataRaftGroupManager] hz.suspicious_ardinghelli.partition-operation.thread-0 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] METADATA CPGroupInfo{id=CPGroupId{name='METADATA', seed=1665975810904, groupId=0}, initialMembers=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], membersCommitIndex=0, members=[RaftEndpoint{uuid='ee97ce6d-c4ac-4510-b57b-4c62831198f3'}, RaftEndpoint{uuid='0c788d37-48e9-490b-9db5-321b2e352092'}, RaftEndpoint{uuid='7bb18b0d-fa03-4604-8d00-72cefb2fb0f5'}], status=ACTIVE} initialization is completed with: [CPMember{uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3, address=[127.0.0.1]:5705}, CPMember{uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5, address=[127.0.0.1]:5701}, CPMember{uuid=0c788d37-48e9-490b-9db5-321b2e352092, address=[127.0.0.1]:5704}]
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [LifecycleService] Thread-75533 - [127.0.0.1]:5703 [dev] [5.3.0-SNAPSHOT] [127.0.0.1]:5703 is SHUTTING_DOWN
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Node] Thread-75533 - [127.0.0.1]:5703 [dev] [5.3.0-SNAPSHOT] Node is already shutting down... Waiting for shutdown process to complete...
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [LifecycleService] Thread-75533 - [127.0.0.1]:5703 [dev] [5.3.0-SNAPSHOT] [127.0.0.1]:5703 is SHUTDOWN
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [LifecycleService] Thread-75533 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] [127.0.0.1]:5702 is SHUTTING_DOWN
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Node] Thread-75533 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Node is already shutting down... Waiting for shutdown process to complete...
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [LifecycleService] Thread-75533 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] [127.0.0.1]:5702 is SHUTDOWN
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [LifecycleService] Thread-75533 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] [127.0.0.1]:5705 is SHUTTING_DOWN
03:05:32,069  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Node] Thread-75533 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Terminating forcefully...
03:05:32,069  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [Node] Thread-75533 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Shutting down connection manager...
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MockServer] Thread-75533 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Removed connection to endpoint: [address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], connection: MockConnection{localEndpoint=[address=[127.0.0.1]:5704, uuid=0c788d37-48e9-490b-9db5-321b2e352092], remoteEndpoint=[address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], alive=false}
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MockServer] Thread-75533 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Removed connection to endpoint: [address=[127.0.0.1]:5704, uuid=0c788d37-48e9-490b-9db5-321b2e352092], connection: MockConnection{localEndpoint=[address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], remoteEndpoint=[address=[127.0.0.1]:5704, uuid=0c788d37-48e9-490b-9db5-321b2e352092], alive=false}
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MockServer] Thread-75533 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Removed connection to endpoint: [address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], connection: MockConnection{localEndpoint=[address=[127.0.0.1]:5701, uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5], remoteEndpoint=[address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], alive=false}
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MockServer] Thread-75533 - [127.0.0.1]:5705 [dev] [5.3.0-SNAPSHOT] Removed connection to endpoint: [address=[127.0.0.1]:5701, uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5], connection: MockConnection{localEndpoint=[address=[127.0.0.1]:5705, uuid=ee97ce6d-c4ac-4510-b57b-4c62831198f3], remoteEndpoint=[address=[127.0.0.1]:5701, uuid=7bb18b0d-fa03-4604-8d00-72cefb2fb0f5], alive=false}
03:05:32,070  WARN |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MembershipManager] Thread-75533 - [127.0.0.1]:5704 [dev] [5.3.0-SNAPSHOT] Member [127.0.0.1]:5705 - ee97ce6d-c4ac-4510-b57b-4c62831198f3 is suspected to be dead for reason: Connection manager is stopped on Member [127.0.0.1]:5705 - ee97ce6d-c4ac-4510-b57b-4c62831198f3 this
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [MembershipManager] Thread-75533 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Removing Member [127.0.0.1]:5705 - ee97ce6d-c4ac-4510-b57b-4c62831198f3
03:05:32,070  INFO |when_newCPMemberIsAddedToTheMetadataGroupAfterRestartAndSnapshot_newMemberInstallsSnapshot| - [PartitionStateManager] Thread-75533 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Storing snapshot of partition assignments wh
...[truncated 2689566 chars]...

```
</details>

--- a/hazelcast/src/test/java/com/hazelcast/cp/internal/CPMemberAddRemoveTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/cp/internal/CPMemberAddRemoveTest.java
@@ -946,6 +946,8 @@ public class CPMemberAddRemoveTest extends HazelcastRaftTestSupport {
         assertEquals(newMetadataGroupId.getSeed(), getRaftService(newInstance1).getMetadataGroupId().getSeed());
         assertEquals(newMetadataGroupId.getSeed(), getRaftService(newInstance2).getMetadataGroupId().getSeed());
 
+        waitUntilCPDiscoveryCompleted(instances[0], newInstance1, newInstance2);
+
         for (int i = 0; i < commitIndexAdvanceCountToSnapshot; i++) {
             getRaftInvocationManager(instances[0]).invoke(getMetadataGroupId(instances[0]), new GetActiveCPMembersOp()).get();
         }

